contracts/tests/test_deposit_locker.py

```diff
@@ -10,55 +10,53 @@
 
 @pytest.fixture()
 def deposit_contract_on_longer_chain(
```

Comment:
The fact that chain is longer is not relevant any more.
Maybe call it `withdrawable_deposit_contract` or `deposit_contract_on_older_chain` or else ...

Reply:
right, I actually wanted to fix that. Will open another PR for it later (since I already based some other work on this one).